api/controllers/user.js

```diff
@@ -0,0 +1,169 @@
+'use strict'
+var express = require('express')
+var router = express.Router()
+var groups = require('../helpers/groups')
+var url = require('url')
+var jwt = require('../helpers/jwt')()
+var db = require('../helpers/db')
+
+/**
+ * @api {get} /users List users 
+ * @apiGroup users
+ * @apiVersion 0.1.0
+ * @apiName listusers
+ *
+ * @apiSuccess {Boolean} OK true if call was successful
+ * @apiSuccess {String[]} errors List of error strings (when `OK` is false)
+ * @apiSuccess {Object} data user data
+ *
+ * @apiSuccessExample Success-Response:
+ *     HTTP/1.1 200 OK
+ * {
+ *   status: "success",
+ *   data: [
+ *   {
+ *     id: 1,
+ *     name: "Tyler",
+ *     age: 3,
+ *     sex: "M"
+ *   }
+ * ],
+ *   message: "Retrieved ALL users"
+ * }
+ *
+ * @apiError NotAuthenticated The user is not authenticated
+ * @apiError NotAuthorized The user doesn't have permission to perform this operation.
+ *
+ */
+router.get('/', function (req, res, next) {
+  console.log("in /user/")
+  // XXX figure out about pagination -- for now, return everything.
```

Comment:
pagination should be pretty straightforward, we just need to track what page of results we're on, and page length, something like:

```db.any('SELECT * from users OFFSET $1 LIMIT $2', [curr_page * page_size, page_size])```